osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -249,7 +249,7 @@ private string getBPMRange(Beatmap beatmap)
                 if (Precision.AlmostEquals(bpmMin, bpmMax))
                     return $"{bpmMin:0}";
 
-                return $"{bpmMin:0}-{bpmMax:0} (mostly {beatmap.ControlPointInfo.BPMMode:0})";
+                return $"{bpmMin:0}-{bpmMax:0} ({beatmap.ControlPointInfo.BPMMode:0})";

```

Comment:
i disagree with this change. we should not remove something just because it doesn't fit. there should first be a discussion and potentially a re-layout. please don't bundle changes like this together.

Reply:
Okay, knew it was a bad idea to change things not related to the PR and with no opinions from other people.